src/main/java/org/jenkinsci/test/acceptance/plugins/ant/AntInstallation.java

```diff
@@ -39,7 +39,9 @@ public AntInstallation(Jenkins context, String path) {
         super(context, path);
     }
 
-    public void useNative() {
-        installedIn(fakeHome("ant", "ANT_HOME"));
+    public String useNative() {
```

Comment:
@oleg-nenashev This is something install to the test scope. I don't see the problem whether it was checked.